pkgs/development/python-modules/radicale-infcloud/default.nix

```diff
@@ -18,6 +19,14 @@ buildPythonPackage {
     hash = "sha256-xzBWIx2OOkCtBjlff1Z0VqgMhxWtgiOKutXUadT3tIo=";
   };
 
+  patches = [
+    # Fix for: https://github.com/Unrud/RadicaleInfCloud/issues/26
```

Comment:
Thanks, but could you rather link the corresponding PR? https://github.com/Unrud/RadicaleInfCloud/pull/27